Sources/TuistGenerator/Generator/ProjectGroups.swift

```diff
@@ -113,6 +112,20 @@ class ProjectGroups {
                              pbxproj: pbxproj)
     }
 
+    func sortMainAndAddDefaultGroups() -> PBXGroup {
+        pbxGroupSorter.sort(with: main)
+        addFirstLevelDefaults(projectGroup: self)
+        return main
+    }
+
+    private func addFirstLevelDefaults(projectGroup _: ProjectGroups) {
```

Comment:
It doesn't look like `projectGroup _: ProjectGroups` is used, it can be omitted :)

Reply:
🤦‍♂️